core/client/src/main/java/alluxio/client/file/options/CompleteFileOptions.java

```diff
@@ -53,12 +53,28 @@ public CompleteFileOptions setUfsLength(long ufsLength) {
     return this;
   }
 
-  /**
-   * @return the name : value pairs for all the fields
-   */
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof CompleteFileOptions)) {
+      return false;
+    }
+    CompleteFileOptions that = (CompleteFileOptions) o;
+    return Objects.equal(mUfsLength, that.mUfsLength);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hashCode(mUfsLength);
+  }
+
   @Override
   public String toString() {
-    return Objects.toStringHelper(this).add("ufsLength", mUfsLength).toString();
+    return Objects.toStringHelper(this)
```

Comment:
I'm doing it this way because it makes it easier to scan visually when more fields are added. The IDE formatting always scrunches up code like this. If you feel strongly I'd be fine with moving the cutoff for abandoning IDE-style to 2+ fields.